Source/ACE.Entity/AceObject.cs

```diff
@@ -1018,25 +1394,7 @@ public void SetStringProperty(PropertyString propertyId, string value)
 
         public Dictionary<PositionType, Position> AceObjectPropertiesPositions { get; set; } = new Dictionary<PositionType, Position>();
 
-        public Position Destination
-        {
-            get { return GetPosition(PositionType.Destination); }
-            set { SetPosition(PositionType.Destination, value); }
-        }
-
-        public Position Location
-        {
-            get { return GetPosition(PositionType.Location); }
-            set { SetPosition(PositionType.Location, value); }
-        }
-
-        public Position Instantiation
-        {
-            get { return GetPosition(PositionType.Instantiation); }
-            set { SetPosition(PositionType.Instantiation, value); }
-        }
-
-        protected Position GetPosition(PositionType positionType)
+        public Position GetPosition(PositionType positionType)
```

Comment:
Why is this (and other get/set property/attribute/position functions) public?  They should be protected, there should be no reason to access them outside of the AceObject (or its subclasses).